### PR TITLE
Convert to python and deploy to GAE standard

### PR DESCRIPTION
**Pull request by breilly2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,51 @@
 # Calhoun
-Notebook preview service
+Jupyter notebook preview service
 
-### Requirements
-- Docker
-- Node 8
-- Google cloud SDK
+### Background
+This project is part of the Terra (né FireCloud) platform of service APIs. It is primarily intended to provide preview versions of notebooks for Terra-based web applications, such as the [Terra workbench](https://www.terra.bio) research environment. It is based on [Calhoun](https://github.com/DataBiosphere/calhoun), a similar service written in Node.js. The service is written wholly in Python 3.7, so that it can be deployed in the Google App Engine Standard environment. Calhoun used Node and shelled out to nbconvert. Since it required both Node and Python, it needed GAE Flexible which lacks FedRAMP certification.
```
Comment (devonbush) [inline README.md]: "It is based on [Calhoun]" is a weird sentence since we're still Calhoun.  maybe just "it was previously written in Node.js

Comment (breilly2) [reply README.md]: Yup. This was in a separate repo before deciding to make a big shift in this existing repo. I'll revise.

Comment (katevoss) [inline README.md]: It's `née`, two ee's.

**Comment by breilly2** [comment]
Currently deployed to dev, alpha, perf, and staging. Will merge and deploy to prod later today.

**Pull request merged**